swing/src/main/java/info/openrocket/swing/gui/configdialog/RocketConfig.java

```diff
@@ -11,11 +11,7 @@
 import java.awt.event.KeyAdapter;
 import java.awt.event.KeyEvent;
 
-import javax.swing.JDialog;
-import javax.swing.JLabel;
-import javax.swing.JOptionPane;
-import javax.swing.JScrollPane;
-import javax.swing.JTextArea;
+import javax.swing.*;
```

Comment:
Please don't use wildcard imports (if you use InteliJ, you can change a setting to disable this automatic behavior)